mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -339,18 +341,18 @@ message GenerateTxRequest {
 
     // List of UnspentTxOuts to be spent by the transaction.
     // All UnspentTxOuts must belong to the same sender_monitor_id.
-    // Mobilecoind would choose a subset of these inputs to construct the transaction.
+    // mobilecoind would choose a subset of these inputs to construct the transaction.
     // Total input amount must be >= sum of outlays + fees.
     repeated UnspentTxOut input_list = 3;
 
     // Outputs to be generated by the transaction. This excludes change and fee.
     repeated Outlay outlay_list = 4;
 
-    // Fee (optional, setting to 0 would cause mobilecoind to choose a fee).
+    // Fee (setting to 0 causes mobilecoind to choose a fee).
     uint64 fee = 5;
 
-    // Tombstone block to use for the transaction. Note that this can later be changed by manipulating
-    // tx_proposal.tx.tombstoneb_clock
+    // Tombstone block to use for the transaction. This can be changed later by manipulating
+    // tx_proposal.tx.prefix.tombstone_block
```

Comment:
This is actually no longer correct, the tombstone block is part of the signature.

Reply:
I see, so the tombstone can no longer be changed post construction. Are there circumstances in which the requested tombstone would be changed (or set) by `mobilecoind` so that the user would need to examine the TxPrefix to see what the transaction would have if it was submitted? I guess this could be the case if somebody built a transaction and then waited a while before sending it and wanted to see how much of the tombstone timeout had already elapsed?

Reply:
I don't understand the question/issue. The tombstone block is set once by mobilecoind when constructing the transaction, either to the one provided by the user or to a one it comes up with on its own.
The user can look at it if they want to, and they can check how many blocks are currently in the ledger.